hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/TestSparkSqlCoreFlow.scala

```diff
@@ -46,18 +47,18 @@ class TestSparkSqlCoreFlow extends HoodieSparkSqlTestBase {
         "COPY_ON_WRITE|false|false|org.apache.hudi.keygen.SimpleKeyGenerator|SIMPLE",
         "COPY_ON_WRITE|true|false|org.apache.hudi.keygen.SimpleKeyGenerator|SIMPLE",
         "COPY_ON_WRITE|true|true|org.apache.hudi.keygen.SimpleKeyGenerator|SIMPLE",
-        "COPY_ON_WRITE|false|false|org.apache.hudi.keygen.NonpartitionedKeyGenerator|GLOBAL_BLOOM",
-        "COPY_ON_WRITE|true|false|org.apache.hudi.keygen.NonpartitionedKeyGenerator|GLOBAL_BLOOM",
-        "COPY_ON_WRITE|true|true|org.apache.hudi.keygen.NonpartitionedKeyGenerator|GLOBAL_BLOOM",
+//        "COPY_ON_WRITE|false|false|org.apache.hudi.keygen.NonpartitionedKeyGenerator|GLOBAL_BLOOM",
+//        "COPY_ON_WRITE|true|false|org.apache.hudi.keygen.NonpartitionedKeyGenerator|GLOBAL_BLOOM",
```

Comment:
can we just remove the comment?

Reply:
i'll check and fix the test

Reply:
The failure is caused by ExpressionPayload not working properly, the fix is a bit involved so tracking it separated in https://issues.apache.org/jira/browse/HUDI-6354. Meanwhile, these testcases should pass with a fix in [1fbc496](https://github.com/apache/hudi/pull/8860/commits/1fbc49655df663197d5bffcd107e0c865a35a953), which bypass the issue. will address the ExpressionPayload issue separately.